pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -94,6 +97,7 @@ func NewReconciler(
 	nodeName types.NodeName,
 	desiredStateOfWorld cache.DesiredStateOfWorld,
 	actualStateOfWorld cache.ActualStateOfWorld,
+	populatorHasAddedPods func() bool,
```

Comment:
just want to double check what is the benefit of using a function instead of passing the pointer to desiredStatePopulator?